tests/test_utils.py

```diff
@@ -211,7 +211,8 @@ def test_pack_query_params_several(self):
             {'apples and oranges': '🍏 & 🍊'},
             {'garbage': ['&', '&+&', 'a=1&b=2', 'c=4&'], 'one': '1'},
             {'&': '&amp;', '™': '&trade;', '&&&': ['&amp;', '&amp;', '&amp;']},
-            # NOTE(vytas): Would fail because of https://github.com/falconry/falcon/issues/1872
+            # NOTE(vytas): Would fail because of
+            # https://github.com/falconry/falcon/issues/1872
             # {'&': '%26', '&&': '%26', '&&&': ['%26', '%2', '%']},
```

Comment:
Hm, it seems this could possibly be uncommented now?

Reply:
good catch!

maybe it makes sense to use another pr though? just to keep things clear?

Reply:
Fine by me

Reply:
I've added https://github.com/falconry/falcon/issues/1940